project/VS2010Express/XBMC.vcxproj

```diff
@@ -372,6 +372,7 @@
     <ClCompile Include="..\..\xbmc\filesystem\MultiPathFile.cpp" />
     <ClCompile Include="..\..\xbmc\filesystem\MusicDatabaseDirectory.cpp" />
     <ClCompile Include="..\..\xbmc\filesystem\MusicDatabaseDirectory\DirectoryNodeGrouped.cpp" />
+    <ClCompile Include="..\..\xbmc\filesystem\MusicDatabaseDirectory\MusicFileItemListModifier.cpp" />
```

Comment:
I'd probably put it into `xbmc/music` or `xbmc/music/windows` as it's specific to the music library representation in our GUI and should not be part of the VFS.